edb/common/parsing.py

```diff
@@ -18,7 +18,8 @@
 
 
 from __future__ import annotations
-from typing import Any, Callable, Tuple, Type, Dict, List, Set, cast
+from typing import Any, Callable, Tuple, Type, Dict, List, Set, \
+                   Optional, cast
```

Comment:
We typically prefer doing continuations with parens, when it is possible, so something like
```
from typing import (
    Any, Callable, Tuple, Type, Dict, List, Set,
    Optional, cast
)
```
would be preferred